test/text/test_cell.py

```diff
@@ -362,3 +361,239 @@ def test_cell_speed_with_long_text():  # issue #907
     assert len(LONG_TEXT_LINES) == 26862
     for line in LONG_TEXT_LINES:
         pdf.cell(0, 3, line, new_x="LMARGIN", new_y="NEXT")
+
+
+def test_cell_border_none(tmp_path):
```

Comment:
Could you regroup all or most of those tests in a single one, with one case per page, in order to limit the number of reference PDF files, please?

All the right/left/top/bottom cases could form a single unit test I think.

`inherit` should have its own dedicated unit test & reference file, but I think more cases should be tested:
* cell `border="INHERIT"` with various table `borders_layout` values
* cell `border="INHERIT"` with table `outer_border_width` set